scripts/analyze_failed_jobs/action_plugins/parsejobs.py

```diff
@@ -0,0 +1,40 @@
+import glob, pickle, buildbot, re, sys, cStringIO as StringIO
+from ansible.utils import template
+from ansible.runner.return_data import ReturnData
+
+class ActionModule:
+    def __init__(self, runner):
+        self.runner = runner
+        
+        
+    def run(self, conn, tmp, module_name, module_args, inject, complex_args=None, **kwargs):
```

Comment:
Same as above, it's the Ansible API... as you see I'm almost not using any of those parameters, this is run by Ansible when the module is called. Maybe I could point people to Ansible docs?